head.go

```diff
@@ -1298,9 +1298,15 @@ func (h *headIndexReader) Series(ref uint64, lbls *labels.Labels, chks *[]chunks
 		if !c.OverlapsClosedInterval(h.mint, h.maxt) {
 			continue
 		}
+		// Set the head chunks as open(being appended to).
+		maxTime := c.maxTime
```

Comment:
Just to check `s.Lock` locks append  /cut to this open chunk here right?

If not, we have a race.

Reply:
yes, I just double checked and the lock is fine